force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +93,25 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<String> wordsListFromSentence = sentence.split(' ');
 
         // Create a set to store the unique words
+        Set<String> uniqueWords = new Set<String>();
 
+        // Loop through the words and add them to the set if they are not already in the set
         // Add each word to the set
+        for (String word : wordsListFromSentence) {
+            if (!uniqueWords.contains(word)) {
```

Comment:
When adding elements to a Set, duplicates are automatically ignored. Therefore, the if statement checking for existence in the set isn't necessary. Although I like that you were making sure! 😄